### PR TITLE
add multi-threshold alert fields + test

### LINKED ISSUE #24: Support for multi-threshold alerts
Currently the provider only supports `type=CLASSIC` alerts.
This issue is about adding support for `type=THRESHOLD` alerts, which allow us to set different thresholds for different levels of severity in a single alert instance.
https://docs.wavefront.com/alerts.html#creating-a-multi-threshold-alert

Enables progress on https://github.com/spaceapegames/terraform-provider-wavefront/issues/42

### PR DESCRIPTION
**Pull request by ashwin1dd**
Closes #24.

- Add additional fields for multi-threshold alerts
- Add unit test

Tests pass locally
Can run the example locally.

### CODE REVIEW

**Comment by ashwin1dd** [comment]
@louism517 @nathmclean Could I get a review on this?

File: alert.go [hunk 1]
```diff
@@ -18,10 +18,12 @@ type Alert struct {
 	AdditionalInfo string `json:"additionalInformation"`
 
 	// Target is a comma-separated list of targets for the Alert
-	Target string `json:"target"`
+	Target  string            `json:"target"`
+	Targets map[string]string `json:"targets"`
```
Comment (louism517) [inline alert.go:27]: Could we just add a comment here explaining that Targets should be used for Threshold alerts? (The godocs are auto-generated from these comments)

File: alert.go [hunk 2]
```diff
@@ -18,10 +18,12 @@ type Alert struct {
...
 
 	// Condition is the condition under which the Alert will fire
-	Condition string `json:"condition"`
+	Condition  string            `json:"condition"`
+	Conditions map[string]string `json:"conditions"`
```
Comment (louism517) [inline alert.go:34]: Sam here, comment.

**Review by louism517 — CHANGES_REQUESTED** [review]
Looks good, just a couple of small comments.

**Comment by ashwin1dd** [comment]
done @louism517

**Comment by ashwin1dd** [comment]
also, do you want me to bump the version? 1.6.0?

**Comment by louism517** [comment]
Yes please! Happy to merge after that. Thanks.

**Comment by ashwin1dd** [comment]
done @louism517, thanks

**Comment by louism517** [comment]
@ashwin1dd Sorry, one more thing. Could you just add a simple example of a threshold alert in examples/alert.go ?

**Comment by ashwin1dd** [comment]
done @louism517. lmk if this looks good. thanks

**Pull request merged**